### ISSUE TITLE
Its really easy to forget to include the cucumber-wire file

### ISSUE THREAD

**Issue by acmcarther** [acmcarther/cucumber-rs#31]
Just spent half an hour debugging my cuke, and the only reason i got it to work was beacuse I suddenly remember that its a thing. Not exactly user friendly.

**Comment by pJunger** [comment]
Could we just generate the wire file somewhere cucumber will see it?

**Comment by acmcarther** [comment]
You mean, like when cuke runs drop the file in, and then when execution is complete we remove it?

We could definitely do that. I wonder if we could somehow ensure that the file gets deleted. In my personal use case, I ctrl-c the test run quite frequently, and that file might get left behind.

Its not a big deal, but I'm imagining someone using this and accidentally committing the wire file. I don't feel very strongly about this, but I'm thinking that #14 might be a more robust solution to this, esp if we get a cargo plugin for code generation.

**Comment by pJunger** [comment]
Yes, at runtime, although I would not remove it, but just regenerate it each time _start_ is called.
It could be set via a _refresh_wire_ method in the create_config chain and thus be optional.
Personally, I would not mind if the generated wire would end up versioned (the current wire is on the vcs as well?), as long as no stale wires are used.

A template or project generator would be really cool, although I think that this is (mostly) orthogonal as it would keep the set address on both sides synchronized. I think this depends on how dynamic these addresses are.

I'm not heavily invested in this, it was just a suggestion that might be easily implemented.
